src/calendar/date-utils.tsx

```diff
@@ -13,3 +13,11 @@ export function monthInMax(year: number, month: number, maxDate?: Date) {
 	}
 	return true;
 }
+
+function stripTime(date: Date) {
```

Comment:
do these need to be in a date-utils file when they're only used in a single location?